### ISSUE TITLE
LET does not work in switch case block

### ISSUE THREAD

**Issue by mrbbp** [espruino/Espruino#1590]
if you add a let declaration in a case block of a switch, Chrome IDE, generate a  **Unexpected token let error**

```
let phrase= "";
function test() {
  let tire = 7;  
  switch (tire) {
    case 7:
      // try a let
      let vTemp = Math.random(); 
      console.log(vTemp);
      break;
    default:
      break;
  }
}
function onInit() {
  setInterval(test,1000);
}
```

**Comment by gfwilliams** [comment]
Forum post at http://forum.espruino.com/conversations/329085/#comment14564236

**Comment by gfwilliams** [comment]
Fixed with 4625d9e9331fb6193c69a34e4ca3ba8f9c7aea58
